libs/types/src/v2.ts

```diff
@@ -94,4 +96,48 @@ export type AttestationElement = StateElement & {
   attestation: Attestation
 }
 
-export type V2FileContractResolutionType = unknown // TODO: replace `unknown` with the actual type
+type V2FileContractResolutionTypeBase = {
+  parent: V2FileContractElement
+}
+
+export type V2FileContractResolutionTypeExpiration =
+  V2FileContractResolutionTypeBase & {
+    type: 'expiration'
+    resolution: Record<string, never> // is this always empty for expiration type?
```

Comment:
@n8maninger could you confirm?

Reply:
That's correct, the resolution on an "expiration" type is empty